configure.ac

```diff
@@ -392,8 +392,7 @@ AS_IF([test -n "$PKG_CONFIG"], [
 					AC_MSG_WARN([[***]])
 					AC_MSG_WARN([[*** Unfortunately, for ease of maintenance, the external libs]])
 					AC_MSG_WARN([[*** are an all or nothing affair.]])
-					AS_ECHO([""])
-					enable_external_libs=no
+					AC_MSG_ERROR(["Unsupported configuration. Will stop build, to prevent silent packaging failures"])
```

Comment:
```suggestion
					AC_MSG_ERROR(["You requested external libraries, but not all were found"])
```

Reply:
Sure
Will do that tomorrow.

It’s time to put the kids to sleep in my TZ and then myself.

Thanks
Alex